qualtran/bloqs/block_encoding/lcu_block_encoding.py

```diff
@@ -108,15 +121,30 @@ def selection_registers(self) -> Tuple[Register, ...]:
 
     @cached_property
     def junk_registers(self) -> Tuple[Register, ...]:
-        return self.prepare.junk_registers
+        return ()
```

Comment:
why this change?

Reply:
This was to get around the OOM issues (This is just block encoding select which doesn't have junk registers, now the circuit widths are the same as before (no ancilla appearing.)